### ISSUE TITLE
05: Don't use logits for 2D projection

### ISSUE THREAD

**Issue by elcorto** [elcorto/2024-thrill-school-machine-learning#9]
Define a CNN which has a final MLP part with > 1 layer and use the linear layer before the logits. See also https://github.com/elcorto/imagecluster .
